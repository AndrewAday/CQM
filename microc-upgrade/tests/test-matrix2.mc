int main(){
	fmatrix fm1;
	fmatrix fm2;



	fm1 = init_fmat_zero(5, 5);
	fm2 = init_fmat_const(2.5, 5, 5);

	print_mat((fm1 + 1.0) + fm2);
	fm1 = fm1 + 1.0;
	print_mat((fm1 + 12.0) .. fm2);
	print_mat(fm1 * fm2);


	return 0;
}
