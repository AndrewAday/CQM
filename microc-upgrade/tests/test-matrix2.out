3.5 3.5 3.5 3.5 3.5
3.5 3.5 3.5 3.5 3.5
3.5 3.5 3.5 3.5 3.5
3.5 3.5 3.5 3.5 3.5
3.5 3.5 3.5 3.5 3.5
162.5 162.5 162.5 162.5 162.5
162.5 162.5 162.5 162.5 162.5
162.5 162.5 162.5 162.5 162.5
162.5 162.5 162.5 162.5 162.5
162.5 162.5 162.5 162.5 162.5
2.5 2.5 2.5 2.5 2.5
2.5 2.5 2.5 2.5 2.5
2.5 2.5 2.5 2.5 2.5
2.5 2.5 2.5 2.5 2.5
2.5 2.5 2.5 2.5 2.5
4.23 4.23
4.23 4.23
4.23 4.23
4.23 4.23
4.23 4.23
4.23 4.23
4.23 4.23
4.23 4.23
4.23 4.23 4.23 4.23 4.23 4.23 4.23 4.23
4.23 4.23 4.23 4.23 4.23 4.23 4.23 4.23
0.41 0.41 0.41 0.41 0.41 0.41 0.41 0.41
0.41 0.41 0.41 0.41 0.41 0.41 0.41 0.41
